brendanbohn/day2.html

```diff
@@ -0,0 +1,56 @@
+<!-- Include Bootstrap in day2.html, and set up a two column layout. 
+
+Inside day2.html, in your left column, use a <ul> to write at least 
+three questions you have about the homework. In the right column, 
+use a <ul> to write three things you think you'll use later.
+
+Use the homework submission instructions from the 22-homework
+or 23-homework repo to submit your homework through a pull request!
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Brendan's Day 2 Homework</title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+	<style type="text/css">
+		h1 {
+			font-family: Helvetica;
+			font-weight: bold;
+			font-size: 100px;
+			margin: 10px 0 30px 15px;
+			color: #4C4C4C;
+		}
+		li {
+			padding: 10px 10px 20px 10px;
+			list-style-type: none;
+			border-bottom: 1px solid gray;
+			font-family: Garamond;
+			font-size: 16px;
+		}
+	</style>
+</head>
+
+<body>
+	<h1>Brendan's Homework: <br> Day 2</h1>
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-4 col-sm-offset-4">
+				<ul>
+					<li>I saw in one of the readings that '!!' could be used to mean 'not not' true; when/how would this be used in practice?</li>
```

Comment:
http://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript  @brendanbohn